functional_tests/test_validator.py

```diff
@@ -18,8 +18,6 @@ def _validator(file_path, for_appversions=None, overrides=None):
     import validator
     import validator.constants
     js = os.environ.get('SPIDERMONKEY_INSTALLATION', 'js')
-    scripting.SPIDERMONKEY_INSTALLATION = js
-    validator.constants.SPIDERMONKEY_INSTALLATION = js
```

Comment:
so `validate()` does the right thing when you pass a custom `spidermonkey`, and that's why you're removing those references to `SPIDERMONKEY_INSTALLATION` ?